<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>AndryBlack</author>
 <class>FontBuilder</class>
 <widget class="QMainWindow" name="FontBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FontBuilder</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_Font">
       <attribute name="title">
        <string>Font</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Font</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="FontSelectFrame" name="frameFontSelect">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="FontOptionsFrame" name="frameFontOptions">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Charaters">
       <attribute name="title">
        <string>Characters</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="CharactersFrame" name="frameCharacters">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Layout">
       <attribute name="title">
        <string>Layout</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Layout engine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxLayouter"/>
        </item>
        <item>
         <widget class="LayoutConfigFrame" name="frameLayoutConfig">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Output">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="OutputFrame" name="frameOutput">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonWriteFont">
          <property name="text">
           <string>Write font</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_Preview">
       <attribute name="title">
        <string>Font image preview</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0" colspan="2">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>264</width>
             <height>393</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="1">
             <widget class="QLabel" name="label_Image">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="lineWidth">
               <number>2</number>
              </property>
              <property name="text">
               <string>image</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBoxDrawGrid">
          <property name="text">
           <string>Draw grid</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_ImageSize">
          <property name="text">
           <string>0x0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_TestFont">
       <attribute name="title">
        <string>Test font</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="FontTestFrame" name="fontTestFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FontSelectFrame</class>
   <extends>QFrame</extends>
   <header>fontselectframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FontOptionsFrame</class>
   <extends>QFrame</extends>
   <header>fontoptionsframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CharactersFrame</class>
   <extends>QFrame</extends>
   <header>charactersframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayoutConfigFrame</class>
   <extends>QFrame</extends>
   <header>layoutconfigframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OutputFrame</class>
   <extends>QFrame</extends>
   <header>outputframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FontTestFrame</class>
   <extends>QFrame</extends>
   <header>fonttestframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>fontParametersChanged()</slot>
  <slot>requestFontsDirChange()</slot>
 </slots>
</ui>
